watch_faces/BinarySystem/source/DataViews/DistanceView.mc

```diff
@@ -0,0 +1,47 @@
+using Toybox.Graphics as Gfx;
+using Toybox.ActivityMonitor as ActMon;
+using Toybox.Application as App;
+using Toybox.WatchUi as Ui;
+
+class DistanceView extends Ui.Drawable
```

Comment:
yeah i was thinking it could be merged into a activity tracking view or something like that since we could potentially add other features as well